docs/source/release_notes/release-notes.rst

```diff
@@ -66,6 +66,11 @@ develop
            transactions had significantly impaired performance if a database node was down.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3199>`__)
 
+    *    - |fixed|
+         - Fixed an issue occurring after a failure to putUnlessExists a commit timestamp that was causing an NPE leading to a confusing error message.
```

Comment:
The release note should capture in which context we would run into this issue in the field.